tools/plugin/modules/ov_noise_suppression/CMakeLists.txt

```
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Noise suppression module
+find_package(OpenVINO COMPONENTS Runtime)
+
+if(OpenVINO_FOUND)
```

```suggestion
if(OpenVINO_FOUND)
```
Can you move the `find_package()` and `if()` to the parent CMakeLists.txt? This avoids the entire file being guarded.